### PR TITLE
Fix design of update error page

### PR DESCRIPTION
**Pull request by jancborchardt**
Before & after:
screenshot from 2018-08-01 13-52-02
screenshot from 2018-08-01 13-49-47
Please review @nextcloud/designers

### CODE REVIEW

File: core/css/guest.css [hunk 1]
```diff
@@ -630,6 +631,9 @@ p.info {
 	margin: 10px;
 	background-size: 32px;
 }
+#update-progress-icon.icon-error-white {
+	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewbox="0 0 16 16" width="16" height="16"><path d="M5.516 2L2 5.516v4.968L5.516 14h4.968L14 10.484V5.516L10.484 2H5.516zM7 4h2v5H7V4zm0 6h2v2H7v-2z" fill="%23fff"/></svg>');
```
Comment (skjnldsv) [inline core/css/guest.css]: Why not relative path url?

Comment (juliushaertl) [reply core/css/guest.css]: I've restored the white icon file in #10459 so you can use it

Comment (jancborchardt) [reply core/css/guest.css]: Ok, so I will restore the white error icon too.

**Review by rullzer — APPROVED** [review]
Fine by me. But somebody with proper css knowledge should have a look as well :P

**Comment by jancborchardt** [comment]
@juliushaertl @skjnldsv fixed your remark, please review again. :)

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**